File: Generala-MF/FMESTADISTICA.vb
Public Class FMESTADISTICA
    'cargo la dll de la generala
    Dim _generalaBLL As New BLL.GENERALA


    Private Sub FMESTADISTICA_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        DGVE.DataSource = Nothing
        DGVE.DataSource = _generalaBLL.listarUSRestadistica

        With DGVE
            .Columns("IDusuario").Visible = False
            .Columns("passwd").Visible = False
            .Columns("nombre").Visible = False
            .Columns("apellido").Visible = False
            .Columns("sexo").Visible = False
            .Columns("email").Visible = False

        End With


    End Sub

    Private Sub Bsalir_Click(sender As Object, e As EventArgs) Handles Bsalir.Click
        Me.Close()
    End Sub
End Class

File: GUI/FMESTADISTICA.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace GUI
{
    public partial class FMESTADISTICA : Form
    {
        public FMESTADISTICA()
        {
            InitializeComponent();
        }

        //cargo la dll de la generala
        BLL.GENERALA _generalaBLL = new BLL.GENERALA();

        private void FMESTADISTICA_Load(object sender, EventArgs e)
        {
            DGVE.DataSource = null;
            DGVE.DataSource = _generalaBLL.listarUSRestadistica();
            //elimino las columnas que no quiero visualizar
            DGVE.Columns["IDusuario"].Visible = false;
            DGVE.Columns["passwd"].Visible = false;
            DGVE.Columns["nombre"].Visible = false;
            DGVE.Columns["apellido"].Visible = false;
            DGVE.Columns["sexo"].Visible = false;
            DGVE.Columns["email"].Visible = false;
        }

        private void Bsalir_Click(object sender, EventArgs e)
        {
            this.Close();
        }

    }
}
